other/materials_designer/specific_examples/passivation_edge_silicon_nanowire.ipynb

```diff
@@ -0,0 +1,330 @@
+{
```

Comment:
<p>phere in which to search for neighbors -&gt; the search sphere for neighbors</p><p> </p><p>The passivation configuration should be per element</p><p> </p><p><code></code><code>py</p><p>from ... import BondDirectionsEnum</p><p> </p><p>element: "Si",</p><p>coordination_threshold: 3,</p><p>max_bonds_to_saturate: 2,</p><p>bond_directions: BondDirectionsEnum.TETRAHEDRAL</p><p></code><code></code></p><p> </p><p>The enum should contain:</p><p> </p><p><code></code><code>py</p><p> </p><p>&nbsp;&nbsp;"tetrahedral": [</p><p>&nbsp;&nbsp;&nbsp;&nbsp;[1, 1, 1], [-1, -1, 1], [1, -1, -1], [-1, 1, -1]</p><p>&nbsp;&nbsp;],</p><p>&nbsp;&nbsp;"octahedral": [</p><p>&nbsp;&nbsp;&nbsp;&nbsp;[1, 0, 0], [-1, 0, 0], [0, 1, 0], [0, -1, 0], [0, 0, 1], [0, 0, -1]</p><p>&nbsp;&nbsp;],</p><p>&nbsp;&nbsp;"planar": [</p><p>&nbsp;&nbsp;&nbsp;&nbsp;[1, 0, 0], [-1, 0, 0], [0, 1, 0]</p><p>&nbsp;&nbsp;]</p><p>}</p><p></code><code></code></p><p> </p><p>We need explain that <code>max_bonds_to_saturate</code> allows to have "non-saturated" </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/173/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>